sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortAggregateExec.scala

```diff
@@ -44,8 +42,7 @@ case class SortAggregateExec(
   with AliasAwareOutputOrdering {
 
   override lazy val metrics = Map(
-    "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
-    "aggTime" -> SQLMetrics.createTimingMetric(sparkContext, "time in aggregation build"))
```

Comment:
unrelated to this PR: the name `aggTime` is a bit misleading. Is it actually the time to build the hash table?

Reply:
@cloud-fan - for hash agg and object hash agg, yes, the `aggTime` is the time to build the hash table. For sort aggregate, there's no proper definition of agg time, as the whole time during sort aggregate is doing aggregate. There's not that useful to measure the execution time for sort aggregate.

Reply:
if there is no group col, does `HashAggregate` always report `aggTime` as 0?

Reply:
> if there is no group col, does HashAggregate always report aggTime as 0?

No, `aggTime` is the time for `HashAggregate` to aggregate all input rows.

Reply:
@cloud-fan - yeah, I agree with you, that `aggTime` is misleading. It's the time to build hash table when group columns exist.